### DISCUSSION TITLE
Delete untagged images after specified delay

### DISCUSSION THREAD

**Discussion by eero-t** [goharbor/harbor#18876] [Ideas]
Tags, especially "latest" one, can move from one image to another, leaving the previous image untagged.

Even when image is referred by its checksum, it's typically a checksum for a (release) tagged image.

IMHO there should be per-repository option for deleting untagged images in it after specified delay.

For older installation it could default to "never", but for new installations, the default could be e.g. 1 week.

**Comment by stonezdj** [comment]
The Garbage collection will remove the untagged image, it depends on your GC schedule, this option is enabled/disabled globally
see: https://goharbor.io/docs/edge/administration/garbage-collection/
```
To Allow garbage collection on untagged artifacts, select the option. If you have this option turned on, the next time Garbage collect runs on your Harbor instance, Harbor will delete untagged artifacts and then perform garbage collection on them.
```

**Comment by eero-t** [comment]
I do not have global Harbour adminstration rights, just admin for specific projects, which are running out of quota.

This is request for having per-project control over this (e.g. to do GC on daily cycle, if global config does it only weekly, or never).
